conans/paths/__init__.py

```diff
@@ -12,7 +12,21 @@
 
 
 def get_conan_user_home():
-    user_home = os.getenv("CONAN_HOME")
+    def _read_user_home_from_rc():
+        try:
+            conanrc_path = os.path.join(os.getcwd(), "conan.conanrc")
+            values = {k: str(v) for k, v in
+                      (line.split('=') for line in open(conanrc_path).read().splitlines() if
+                       not line.startswith("#"))}
+            conan_home = values["conan_home"]
+            # check if it's a local folder
+            if conan_home[:2] in ("./", ".\\") or conan_home.startswith(".."):
+                conan_home = os.path.abspath(os.path.join(os.getcwd(), conan_home))
```

Comment:
The home should be computed relative to the folder containing the ``.conanrc``, not ``os.getcwd()``, otherwise it will be confusing.

Reply:
Yes, that was the original idea. Too often you are not in the root, but in a ``build`` subfolder or something.